src/libraries/Common/src/System/Security/Cryptography/MLKem.cs

```diff
@@ -1106,6 +1162,122 @@ public byte[] ExportEncryptedPkcs8PrivateKey(string password, PbeParameters pbeP
             return ExportEncryptedPkcs8PrivateKey(password.AsSpan(), pbeParameters);
         }
 
+        /// <summary>
+        ///  Exports the current key in a PEM-encoded representation of the PKCS#8 EncryptedPrivateKeyInfo
+        /// representation of this key, using a byte-based password.
+        /// </summary>
+        /// <param name="passwordBytes">
+        ///   The bytes to use as a password when encrypting the key material.
+        /// </param>
+        /// <param name="pbeParameters">
+        ///   The password-based encryption (PBE) parameters to use when encrypting the key material.
+        /// </param>
+        /// <returns>
+        ///   A byte array containing the PKCS#8 PrivateKeyInfo representation of the this key.
```

Comment:
Neither a byte[] nor PrivateKeyInfo.

```suggestion
        ///   A string containing the PEM-encoded PKCS#8 EncryptedPrivateKeyInfo.
```